js/pricing.js

```diff
@@ -7,27 +10,145 @@
+            apiUrl = "{{site.data.payment_config.devApiUrl}}";
+        } else {
+            apiUrl = "{{site.data.payment_config.prodApiUrl}}";
+        }
+
+        $consentCheckboxes.change(() => {
+            changeElementState($orderButton, isConsentObtained());
+        });
+
+        $orderButton.click(() => {
+            if (isConsentObtained()) {
+                submitOrder();
+            }
         });
 
-        $confirmDevelopmentAgreement.change(function () {
-            confirmAgreement(this, $disabledButton);
+        $linkBack.click(e => {
+            e.preventDefault();
+            hideRedirect();
         });
 
-        function confirmAgreement(element, disabledElement) {
-            const isConfirmed = $confirmPersonalInformation.prop('checked') && $confirmDevelopmentAgreement.prop('checked');
-            if (isConfirmed) {
-                disabledElement.removeAttr('disabled');
-                disabledElement.removeClass('disabled');
+        /**
+         * Checks if all consent checkboxes are checked.
+         *
+         * @return {boolean} true if all checkboxes are checked
+         */
+        function isConsentObtained () {
+            return $consentCheckboxes.length == $consentCheckboxes.filter(":checked").length;
+        }
+
+        /**
+         * Changes element disabled/enabled state.
+         *
+         * @param {jQuery} element the element, which state will be changed
+         * @param {boolean} enabled if state is `true` the element will be enabled, otherwise it
+         * will be disabled
+         */
+        function changeElementState(element, enabled) {
+            if (enabled) {
+                element.removeAttr('disabled');
+                element.removeClass('disabled');
             } else {
-                disabledElement.attr('disabled', true);
-                disabledElement.addClass('disabled');
+                element.attr('disabled', true);
+                element.addClass('disabled');
             }
         }
+
+        /**
+         * Submit order handler.
+         */
+        function submitOrder() {
+            const privacyConsent = $privacyConsent.prop("checked");
+            const supportAgreementConsent = $supportAgreementConsent.prop("checked");
+            const data = JSON.stringify({privacyConsent, supportAgreementConsent});
+            const transactionUrl = `${apiUrl}/transaction`;
+            sendPaymentTransaction(transactionUrl, data);
+        }
+
+        /**
+         * The user's consent.
+         *
+         * <p>Contains user's consents for services and data processing flows.
+         *
+         * @typedef {Object} Consent
+         * @property {boolean} privacyConsent indicates whether the user give a consent to
+         * process his personal information
+         * @property {boolean} supportAgreementConsent indicates whether the user agree to be bound by the
+         * terms of Development Support Agreement
```

Comment:
Please consider applying fixes for the data types that are already commented on in the backend PR.